mathesar_ui/src/systems/table-view/table-inspector/CollapsibleHeader.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { Dropdown, Icon } from '@mathesar/component-library';
+  import { iconDatabase } from '@mathesar/icons';
+
+  export let title: string;
+  export let isDBLevelConfiguration = false;
```

Comment:
Instead of `isDBLevelConfiguration`, we should be using `isDbLevelConfiguration`, per our code standards to treat acronyms as words. Note that we still have some older code (e.g. `postAPI`) which we wrote before adopting that code standard, but it would be nice to try adhering to this standard going forward.

Reply:
Looks good. Thanks.